### PR TITLE
release fixes

### PR DESCRIPTION
**Pull request by FBartos**
fixes: https://github.com/jasp-stats/jasp-test-release/issues/1972
fixes: https://github.com/jasp-stats/jasp-test-release/issues/1974
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2016
fixes: https://github.com/jasp-stats/jasp-test-release/issues/2015

Can't really fix it as developmental modules don't install in Windows...

### CODE REVIEW

File: R/summarystatsbayesianztest.R [hunk 1]
```diff
@@ -212,6 +212,14 @@ SummaryStatsBayesianZTest <- function(jaspResults, dataset = NULL, options, ...)
       robustnessPlot$setError(gettext("Bayes factor contours breaks could not be parsed into numbers."))
       return()
     }
+    if (any(bfBreaks <= 0)) {
+      robustnessPlot$setError(gettext("Bayes factor contours breaks must be positive."))
+      return()
+    }
+    if (!all(order(bfBreaks) == seq_along(bfBreaks))) {
```
Comment (vandenman) [inline R/summarystatsbayesianztest.R:219]: perhaps?
```suggestion
    if (is.unsorted(bfBreaks, strictly = TRUE)) {
```

```r
is.unsorted(1:5) # FALSE
is.unsorted(5:1) # TRUE
is.unsorted(c(1, 1:5), strictly = FALSE) # FALSE
is.unsorted(c(1, 1:5), strictly = TRUE) # TRUE
```

File: inst/qml/SummaryStatsABTestBayesian.qml [hunk 2]
```diff
@@ -30,15 +30,15 @@ Form
 	Group
 	{
 		title: qsTr("Group 1")
-		IntegerField { name: "y1";	label: qsTr("Successes")	}
-		IntegerField { name: "n1";	label: qsTr("Sample Size")	}
+		IntegerField { name: "y1";	label: qsTr("Successes"); max = n1.value}
```
Comment (vandenman) [inline inst/qml/SummaryStatsABTestBayesian.qml:33]:
```suggestion
		IntegerField { name: "y1";	label: qsTr("Successes"); max: n1.value}
```
QML uses `:` and not `=` :wink:

File: R/summarystatsbayesianztest.R (hunk 1 above)
Comment (FBartos) [reply R/summarystatsbayesianztest.R]: nice, didn't know about this function!

File: inst/qml/SummaryStatsABTestBayesian.qml (hunk 2 above)
Comment (FBartos) [reply inst/qml/SummaryStatsABTestBayesian.qml:33]: haha, thanks. Those are exactly the things that happen when you can't install the module to test the code....

**Review by vandenman — APPROVED** [review]
All issues are fixed

**Pull request merged**